Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -572,8 +568,9 @@ theorem repr_mul : ∀ (o₁ o₂) [NF o₁] [NF o₂], repr (o₁ * o₂) = rep
         mul_omega_dvd (natCast_pos.2 n₁.pos) (nat_lt_omega _)]
       simpa using opow_dvd_opow ω (one_le_iff_ne_zero.2 this)
 
-/-- Calculate division and remainder of `o` mod ω.
-  `split' o = (a, n)` means `o = ω * a + n`. -/
+/-- Calculate division and remainder of `o` mod `ω`.
```

Comment:
Better would be `Calculate division and remainder   : split ... means ...` (i.e., with a colon instead of a period)